.github/workflows/build-and-test.yml

```diff
@@ -321,18 +356,14 @@ jobs:
           popd
 
       - name: Run cuda.core tests
-        shell: bash --noprofile --norc -xeuo pipefail {0}
         run: |
-          if [[ ${{ matrix.python-version }} == "3.13" ]]; then
-            # TODO: remove this hack once cuda-python has a cp313 build
-            if [[ $SKIP_CUDA_BINDINGS_TEST == 1 ]]; then
```

Comment:
Note: this hack can be removed now that we generate Python 3.13 wheels for `cuda.bindings` 11.8 and can retrieve them in the CI; we do not need them published on PyPI in order to use them!